Silicon/CommonSocPkg/Library/HeciLib/HeciCore.h

```diff
@@ -0,0 +1,20 @@
+/** @file
+
+  Copyright (c) 2018 - 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _HECI_CORE_H_
+#define _HECI_CORE_H_
+
+#include <Library/PciLib.h>
+
+#define HeciPciRead32(func, reg)     PciRead32 (PCI_LIB_ADDRESS(ME_BUS, ME_DEVICE_NUMBER, func, reg))
+#define HeciPciRead16(func, reg)     PciRead16 (PCI_LIB_ADDRESS(ME_BUS, ME_DEVICE_NUMBER, func, reg))
+#define HeciPciOr16(func, reg, val)  PciOr16 (PCI_LIB_ADDRESS(ME_BUS, ME_DEVICE_NUMBER, func, reg), val)
+#define HeciPciOr8(func, reg, val)   PciOr8 (PCI_LIB_ADDRESS(ME_BUS, ME_DEVICE_NUMBER, func, reg), val)
+#define HeciPciAnd8(func, reg, val)  PciAnd8 (PCI_LIB_ADDRESS(ME_BUS, ME_DEVICE_NUMBER, func, reg), val)

```

Comment:
If you want to use these macros, you need to update all files to use these. The common heci library is using MeGetHeciMmPciAddress() and Mmio() APIs without using any Pci() APIs.

Reply:
maybe better to avoid using hardcoded ME_DEVICE_NUMBER and ME_BUS.   Using MeGetHeciMmPciAddress（）is more portable.

Reply:
Done